admin/src/app/foms/interactions/interaction-detail/interaction-detail.form.ts

```diff
@@ -10,7 +11,7 @@ export class InteractionDetailForm implements Pick<InteractionRequest, typeof UP
 
   @required()
   @prop()
-  communicationDate: string = '';
+  communicationDatePickerDate: Date; // for datePicker only
```

Comment:
Why is this needed? Maybe expand on this comment a bit.

Reply:
Hi Basil, I renamed this field and comment is added for more detail.
When replacing library `moment` with 'luxon`, I found previously moment was TOO smart that it was both good with init with string type and date type. However, when switching to 'luxon', it did not work anymore for this field. So I decided to rename this field and specifically gave it a 'Date' type, that's what datePicker will set and to avoid confusion with setting to 'body' field for 'communicationDate' which only accepts 'string' type.